js/containers/buttons/SaveDeployPlan.jsx

```diff
@@ -0,0 +1,24 @@
+var ReactRedux = require('react-redux');
+
+var _ = require('underscore');
+var actions = require('../../_actions.js');
+var Button = require('../../components/Button.jsx');
+
+const mapStateToProps = function(state) {
+	return {
+		display: !!!state.deployment.id,
```

Comment:
Isn't there a better way of casting these to booleans? :) `!!!!!!!!!!!!!!!!!!!!`